src/Sentry.AspNet/HttpContextExtensions.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections.Specialized;
+using System.Web;
+
+namespace Sentry.AspNet
+{
+    /// <summary>
+    /// Sentry extensions for <see cref="HttpContext"/>.
+    /// </summary>
+    public static class HttpContextExtensions
+    {
+        private static SentryTraceHeader? TryGetTraceHeader(NameValueCollection headers)
+        {
+            try
+            {
+                var traceHeader = headers.Get(SentryTraceHeader.HttpHeaderName);
+                return SentryTraceHeader.Parse(traceHeader);
```

Comment:
Next up is doing a TryParse because we can'ta ssume every request will come from an instrumented SDK so this will likely happen a lot (throw)